package.json

```diff
@@ -9,6 +9,7 @@
     "connected-react-router": "^6.6.0",
     "fetch-send-json": "0.0.2",
     "js-sha256": "^0.9.0",
+    "moment": "^2.24.0",
```

Comment:
https://bundlephobia.com/result?p=moment@2.24.0 feels a bit heavy if we only going to use for time stamps.  Maybe worth checking alternatives?

Reply:
Changed to use: [](https://bundlephobia.com/result?p=timeago.js@4.0.1)

Reply:
Changed to use: https://bundlephobia.com/result?p=timeago.js@4.0.1